src/DynamoCoreWpf/Utilities/NodeContextMenuBuilder.cs

```diff
@@ -446,7 +446,7 @@ internal static MenuItem BuildHelpMenuItem()
             Properties.Resources.NodeContextMenuShowLabels,
             Properties.Resources.NodeContextMenuRenameNode,
             Properties.Resources.ContextMenuLacing,
-            //Wpf.Properties.Resources.NodeInformationalStateDismissedAlerts,
+            Properties.Resources.NodeInformationalStateDismissedAlerts,
```

Comment:
This fixes a bug whereby Dismissed Alerts could appear in the node context menu twice.
I simply forgot to uncomment this a while back.